lms/envs/common.py

```diff
@@ -408,6 +408,9 @@
 
     # Enable LTI Provider feature.
     'ENABLE_LTI_PROVIDER': False,
+
+    # Enable automatic status emails for ICRV attempts
+    'ENABLE_ICRV_STATUS_EMAILS': False,
```

Comment:
Thank you @clintonb. @tasawernawaz please use a Waffle switch to toggle this feature.